components/alibi-detect-server/adserver/cd_model.py

```diff
@@ -5,6 +5,55 @@
 from .numpy_encoder import NumpyEncoder
 from alibi_detect.utils.saving import load_detector, Data
 from adserver.base import AlibiDetectModel
+from seldon_core.user_model import SeldonResponse
+
+
+def _drift_to_metrics(drift):
+    metrics = []
+
+    batch_score = drift.get("batch_score")
+    if batch_score:
+        metrics.append(
+            {
+                "key": "seldon_metric_drift_batch_score",
```

Comment:
Are these metrics keys not tied to a model or deployment? What happens if there are 2 in the same namespace. Any clashes?

Reply:
These are made unique by tags, which are added by the SeldonMetrics object - similar to the latency / request per second metrics

Reply:
To be more specific, this uses the env variables that are set in the container (similar to the custom metrics server example), so you would have PREDICTOR_ID, etc.